error/error_handler.go

```diff
@@ -114,6 +112,18 @@ func EchoHTTPErrorHanderFunc(e error, c echo.Context) (bool, error) {
 			})
 		}
 
+	case http.StatusServiceUnavailable:
```

Comment:
Let's use `http.StatusGatewayTimeout` instead of  `http.StatusServiceUnavailable`

Reply:
I have read some documentation about explanations for 503 and 504 errors. They define a situation where a typical 503 error can be recovered, but a 504 error cannot. Therefore, I think that timeouts in a service should be classified as a type of 503 error. Perhaps I am wrong, and I'm not certain.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504